src/ripple/core/impl/Config.cpp

```diff
@@ -454,7 +454,7 @@ Config::loadFromString(std::string const& fileContents)
         if (boost::iequals(strTemp, "none"))
             FETCH_DEPTH = 0;
         else if (boost::iequals(strTemp, "full"))
-            FETCH_DEPTH = 1000000000u;
+            FETCH_DEPTH = 1'000'000'000u;
```

Comment:
Consider [`std::numeric_limits<std::uint32_t>::max`](https://en.cppreference.com/w/cpp/types/numeric_limits/max) instead for `FETCH_DEPTH` and `LEDGER_HISTORY`.

Reply:
Do you know if there was a reason this was defined as 1 billion instead of `max()` in the first place?

Reply:
There was no particular reason.